schemas/io.snowplow.ecomm/cart/jsonschema/1-0-0

```diff
@@ -0,0 +1,166 @@
+{
+    "description": "The cart entity contains the products and bundles that have been added to cart, as well as information about discounts and total costs",
+    "properties": {
+        "cart_id": {
+            "type": "string",
+            "maxLength": 255
+        },
+        "total_items": {
+            "type": [
+                "integer",
+                "null"
+            ],
+            "description": "Total number of items in the cart.",
+            "minimum": 0,
+            "maximum": 2147483647
+        },
+        "subtotal": {
```

Comment:
I feel the `subtotal`, `shipping_cost` and `tax` belong in the `/order/` schema. Am I right ?

Reply:
We think for purpose of analysing behaviour with events before an order is created it'll be helpful to have these fields with the cart entity, so worth to leave them in here as well as on the order/payment entities.